l10n_it_website_portal_fatturapa_sale/i18n/it.po

```diff
@@ -0,0 +1,26 @@
+# Translation of Odoo Server.
+# This file contains the translation of the following modules:
+#	* l10n_it_website_portal_fatturapa_sale
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: Odoo Server 12.0+e\n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2019-07-23 10:40+0000\n"
+"PO-Revision-Date: 2019-07-23 10:40+0000\n"
+"Last-Translator: Simone Rubino <simone.rubino@agilebg.com>\n"
+"Language-Team: \n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: \n"
+"Plural-Forms: \n"
+
+#. module: l10n_it_website_portal_fatturapa_sale
+#: model:ir.model,name:l10n_it_website_portal_fatturapa_sale.model_res_partner
+msgid "Contact"
+msgstr "Contatto"
+
+#. module: l10n_it_website_portal_fatturapa_sale
+#: model_terms:ir.ui.view,arch_db:l10n_it_website_portal_fatturapa_sale.payment_tokens_list
+msgid "I request to receive electronic invoice"
+msgstr "Voglio ricevere la fattura elettronica"
```

Comment:
In base a  
https://github.com/OCA/l10n-italy/pull/871#discussion_r270635663

direi 
Richiedi/o invio fattura elettronica

Reply:
grazie @primes2h, fatto